packages/ember-metal/lib/meta.js

```diff
@@ -33,7 +32,7 @@ if (DEBUG) {
 @module ember-metal
 */
 
-export const UNDEFINED = symbol('undefined');
```

Comment:
It's not a string though, right? It's a unique object with a string description.  I think we are probably good to leave this as Symbol for future interpreter optimization.

Granted, the transpile may do string compare now, but we shouldn't write code based on how it's transpiled.